apps/notifications/webpack.config.js

```diff
@@ -65,6 +65,15 @@ function getWebpackConfig(
 			},
 			watchFiles: [ 'dist/**/*' ],
 		},
+		resolve: {
+			...webpackConfig.resolve,
+			alias: {
```

Comment:
This performs the bulk of the aliasing, and is [straight from the Preact docs](https://preactjs.com/guide/v10/getting-started#aliasing-in-webpack).